Introduction of value layers (#722)
flowforge/website: src/handbook/product/pricing.md

# Pricing Principles

> This page sets out the concepts that we sell and how those are measured across both SaaS and Licensed Editions of the platform. This does not define specific dollar amounts for any items, those are commercial decisions outside the scope of this document.

Reviewer: ```suggestion
This page sets out the concepts that we license and what units are measured across both FlowForge Cloud  and Self-managed. Commercial decisions outside the scope of this document.
```

We have three primary tiers: Open-Source (Starter), Premium, and Enterprise. The value and features provided within each tier correspond to the specific [persona](./personas/index.md), acknowledging that a higher placement in the organizational chart typically implies diffrent requirements.

## Layers

| Tier                  | Objective                                         | Problem it Solves                                               | Persona                                        |
| --------------------- | ------------------------------------------------- | --------------------------------------------------------------- | ---------------------------------------------- |

> | Open-Source (Starter) | Introduce Node-RED                                | - Hosting Node-RED<br> - TLS/SSL<br> - Subdomain<br> - Backups (snapshots) | - Individual contributor<br> - PLC engineer<br> - Line worker |

Reviewer: ```suggestion
| Starter (Open-source) | Introduce Node-RED                                | - Hosting Node-RED<br> - TLS/SSL<br> - Subdomain<br> - Backups (snapshots) | - Individual contributor<br> - PLC engineer<br> - Line worker |
```

| Premium               | Scale the roll-out of Node-RED                    | - Managing more than 5 Node-RED instances<br> - Security<br> - Effective collaboration  | - IIoT manager<br> - Plant manager            |
| Enterprise            | Node-RED as a company standrad                    | - Auditability<br> - Enterprise intergration                | - Central IT departments<br> - (Plant manager) |

## Teams

Teams are the basic unit within FlowForge Cloud and the entity to which bills are charged.

> A teams can create zero or more applications, and 1 or more user can be members.

Reviewer: ```suggestion
A teams can create zero or more applications, and one or more user can be members.
```

Devices are part of the licensed instances. The limit in the open source edition is 50 devices.

## Licenses

> From the perspective of licensing the open source and FlowForge Premium, and FlowForge Enterprise are different tiers of the same product.

Reviewer: ```suggestion
From the perspective of licensing the there's an unlicensed edition: open-source. This version is always self-managed. There's also a licensed version of FlowForge, this can be self-managed or FlowForge Managed. For FlowForge managed properties there's 3 tiers; Starter, Premium, and Enterprise.
```

The open source edition doesn't require a license key to be uploaded. Without a valid license a basic set of features and

> quantity of projects(50), users(150), teams(50), and devices(50) are available.

Reviewer: ```suggestion
quantity of instances(50), users(150), teams(50), and devices(50) are available.
```
